cluster/sbin/snappy-server.sh

```diff
@@ -36,11 +37,43 @@ shift
 . "$SNAPPY_HOME/bin/load-spark-env.sh"
 . "$SNAPPY_HOME/bin/load-snappy-env.sh"
 
-
+noOfInputsArgs=$#
 
 # Start up  the server instance
 function start_instance {
   "$SNAPPY_HOME"/bin/snappy server "$mode" "$@"
 }
 
+if [ $noOfInputsArgs -le 1 ]
```

Comment:
Same concern as above

Reply:
Resolved: Modified the code, now as per the current modification, if the user doesn't provide -dir argument, ask the user to provide it.